### PR TITLE
[author][new] Add Open Location Code

### PR DESCRIPTION
**Pull request by drinckes**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/google/open-location-code
 * Official website (optional, not the repository): http://openlocationcode.com
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 683
   - Count of watchers: 70
   - Count of forks: 86
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [X] I'm the author of this library
   * [X] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Hi @drinckes, is it possible to use auto-update config here? Thanks.

File: ajax/libs/openlocationcode/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "library",
+    "openlocationcode"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/google/open-location-code.git",
+    "basePath": "js/src",
+    "files": [
+      "openlocationcode-min.js",
+      "openlocationcode.js"
```
Comment (pvnr0082t) [inline ajax/libs/openlocationcode/package.json]: @drinckes Let's use `openlocationcode*` here.

**Comment by drinckes** [comment]
Right you are. :-)

File: ajax/libs/openlocationcode/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/openlocationcode/package.json]: Or something like `"openlocationcode*.+(js|map)"`

**Pull request merged**